### PR TITLE
Pd 996 rework and publish solutions team entra ds azure ad ds article

### PR DESCRIPTION
**Pull request by DjP-iX**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/Solutions/Integrations/EntraID.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+3. Join the domain.
+
+   a. Go to **Credentials > Directory Services** and click **Configure Active Directory** to open the **Active Directory** screen.
+
+   {{< trueimage src="/images/SCALE/Credentials/ActiveDirectoryBasicOptions.png" alt="Active Directory Screen" id="Active Directory Screen" >}}
+
+   b. Enter the Entra domain name in **Domain Name** and the administrator account credentials in **Domain Account Name** and **Domain Account Password**.
+
+   c. Select **Enable** to attempt to join the AD domain immediately after saving the configuration.
+     SCALE populates the **Kerberos Realm** and **Kerberos Principal** fields on the **Advanced Options** settings screen.
```
Comment (william-gr) [inline content/Solutions/Integrations/EntraID.md:93]: This last line seems unnecessary.

File: content/Solutions/Integrations/EntraID.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+   d. Click **Save**.
+
+      TrueNAS can take a few minutes to populate directory information after configuration.
+      To check the join progress, open the <i class="material-icons" aria-hidden="true" title="Assignment">assignment</i> **Task Manager** in the upper-right corner.
+      TrueNAS displays any errors during the join process in the **Task Manager**.
+
+      When the import completes, domain users and groups become available.
```
Comment (william-gr) [inline content/Solutions/Integrations/EntraID.md:101]: Its not an import, its just joining.

File: content/Solutions/Integrations/EntraID.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+4. Verify you are able to lookup domain users.
+
+   From a shell session, enter the command {{< cli >}}wbinfo -u{{< /cli >}}.
```
Comment (william-gr) [inline content/Solutions/Integrations/EntraID.md:105]: Do not need to refer to the shell. Can use the UI to populate AD users, e.g. Permissions UI for a dataset.

**Review by Mrt134 — APPROVED** [review]
Build test looks great and have offline stakeholder approval on the content - good job!

**Pull request merged**